src/tests/unit/tests/injected/components/command-bar.test.tsx

```diff
@@ -121,13 +135,11 @@ describe('CommandBar', () => {
                 onClickInspectButton: onClickMock.object,
             };
 
-            const wrapper = shallow(<CommandBar {...props} />);
-
-            const button = wrapper.find('.insights-dialog-button-inspect');
+            render(<CommandBar {...props} />);
 
-            button.simulate('click', eventStub);
+            getMockComponentClassPropsForCall(DefaultButton).onClick();
 
-            onClickMock.verify(onClick => onClick(eventStub), Times.once());
+            onClickMock.setup(onClick => onClick(eventStub)).verifiable(Times.once());
```

Comment:
This can stay the same as it was.
```suggestion
            onClickMock.verify(onClick => onClick(eventStub), Times.once());
```